src/video/dedicated_v.h

```diff
@@ -28,6 +28,19 @@ class VideoDriver_Dedicated : public VideoDriver {
 	bool ToggleFullscreen(bool fullscreen) override;
 	const char *GetName() const override { return "dedicated"; }
 	bool HasGUI() const override { return false; }
+
+	std::chrono::steady_clock::duration GetGameInterval() override
+	{
+		/* Infinite speed, as quickly as you can. */
+		if (_game_speed == 0) return std::chrono::microseconds(0);
+
+		long min = MILLISECONDS_PER_TICK * 1000;
+		long ms = 1000000 / _settings_client.gui.refresh_rate;
```

Comment:
You seem to mix up two concepts of the game. We have two threads: draw-thread and game-thread. The first is limited by `refresh_rate`, and often expressed in Hz. Like 60Hz for most people these days. The second is limited by `MILLISECONDS_PER_TICK`, which is the same for every game and (currently) not changable. This is set to 27ms per tick.

This change mixes up those two concepts. Please don't do that.